File: Plugins/Aspose_Diagram_Java_for_Ruby/lib/asposediagramjava/Shapes/changeshapeposition.rb
module Asposediagramjava
  module ChangeShapePosition
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'

        # Create instance of Diagram
        diagram = Rjb::import('com.aspose.diagram.Diagram').new(data_dir + "Drawing.vsd")

        shapes = diagram.getPages().getPage(0).getShapes()
        
        i = 0
        while i < shapes.getCount()
            shape = shapes.get(i)
            if shape.getNameU() == "Process" && shape.getID() == 2
                shape.move(1, 1)
            end
            i +=1
        end
        
        # Save diagram
        diagram.save(data_dir + "ChangeShapePosition.vdx", Rjb::import('com.aspose.diagram.SaveFileFormat').VDX)

        puts "Changed postion of a shape."
    end
  end
end

File: Plugins/Aspose_Diagram_Java_for_PHP/src/aspose/diagram/WorkingwithShapes/ChangeShapePosition.php
<?php
namespace Aspose\Diagram\WorkingwithShapes;

use com\aspose\diagram\Diagram as Diagram;
use com\aspose\diagram\SaveFileFormat as SaveFileFormat;
class ChangeShapePosition{

    public static function run($dataDir=null){

        # Create instance of Diagram
        $diagram = new Diagram($dataDir."Drawing.vsd");

        $shapes=$diagram->getPages()->getPage(0)->getShapes();
        $i = 0;
        while ($i<(int)(string)$shapes->getCount()) {
        $shape = $shapes->get($i);
        if ($shape->getNameU() == "Process" && (int)(string)$shape->getID() == 2) {
        $shape->move(1, 1);
        }
        $i += 1;
        }

        # Save diagram
        $saveFileFormat=new SaveFileFormat();
        $diagram->save($dataDir."ChangeShapePosition.vdx",$saveFileFormat->VDX);
        print"Changed postion of a shape.".PHP_EOL;
    }
}